qna/AT-corona-USt-Senkung-Gastronomie-2020.md

```diff
@@ -12,54 +12,42 @@ lang-de, market-at, middleware, PosCreator, PosDealer, Consultant
 
 ## Answer
 
-### A. Information der WKO UBIT, 19.06.2020
+### Information des BMF betreffend die Umsatzsteuersenkung
+Zur Unterstützung der Gastronomie, der Kulturbranche sowie des Publikationsbereichs, die von der COVID-19-Krise in einem besonderen Ausmaß betroffen sind, soll zusätzlich zu den bisher getroffenen Maßnahmen, befristet vom 1.7.2020 bis 31.12.2020 ein ermäßigter Umsatzsteuersatz iHv 5% eingeführt werden.
```

Comment:
Ich habe heute in den Nachrichten gehört, dass auch Reparaturen an KFZ mit einem verminderten UST-Satz gestützt werden.

Reply:
Korrekt - Die Info wurde und wird laufend aktualisiert.

Reply:
Schreiben wir das dazu: Bitte schauen Sie regelmäßig auf diese Seite, da die Bundesregierung die Möglichkeiten der Unterstützung laufend den wirtschaftlichen Anforderungen anpasst.